### ISSUE TITLE
[Label] Space between labels after line-breaks missing

### ISSUE THREAD

**Issue by mkrecek234** [fomantic/Fomantic-UI#2986]
# Bug Report
If you have multiple labels after one another, and a soft-wrap/line-break happens, labels touch each other at the bottom/top side. There should be space in between for better layout.

## Steps to reproduce
Just make multiple labels after one another, so the browser makes a line-break.

## Expected result
Space in between labels vertically.

## Actual result
No space

## Screenshot (if possible)
![image](https://github.com/fomantic/Fomantic-UI/assets/28671486/337737e9-b1d1-4e10-84df-89de09157948)

Sample page: https://ui.agiletoolkit.org/demos/collection/lister-ipp.php

## Version
Latest

**Comment by lubber-de** [comment]
Use the grouping feature, which already got a vertical margin

See https://jsfiddle.net/lubber/matb5os1/2/
![image](https://github.com/fomantic/Fomantic-UI/assets/18379884/3049e858-fdfd-4cb7-ac4e-8d3bff4a59f3)

**Comment by lubber-de** [comment]
I enhanced the docs to clarify that by https://github.com/fomantic/Fomantic-UI-Docs/pull/488

**Comment by mkrecek234** [comment]
@lubber-de Thanks a lot, that helps!

**Comment by Teraskull** [comment]
Grouping with the `labels` class does not add the line-break if all of them are image labels:

![image](https://github.com/user-attachments/assets/e10274be-1dc9-466f-b676-196e3d00c4fc)

```html
<div class="ui labels">
    <div class="ui image label">
        <img src="clear.png" class="ui image">
        <span>Clear</span>
    </div>
    <div class="ui image label">
        <img src="clear.png" class="ui image">
        <span>Clear</span>
    </div>
</div>
```

**Comment by lubber-de** [comment]
@Teraskull  Fixed by #3110
